### ISSUE TITLE
OrderWatcher stopped working with mainnet

### ISSUE THREAD

**Issue by imbaniac** [0xProject/0x-monorepo#646]
## Expected Behavior

OrderWatcher to not throw error after initializing with zeroEx.createOrderWatcherAsync

## Current Behavior

`(node:50398) UnhandledPromiseRejectionWarning: Error: Invalid JSON RPC response: ""
    at Object.InvalidResponse (/Users/timreznik/code/instex-api/node_modules/web3-core-helpers/src/errors.js:42:16)`

## Steps to Reproduce (for bugs)

```

1. Use mainnet
2. zeroEx.createOrderWatcherAsync({
    cleanupJobIntervalMs: 60000,
  });
```

## Context

Everything works with testrpc.
After commenting out OrderWatcher everything works, thus it isn't node connection failure.

## Your Environment

|             Package | Version |
| ------------------: | :------ |
|             `0x.js` | 0.38.3  |
|             `web3` | 1.0.0-beta.33  |
| `Exchange Contract` | v1      |
| `Parity` | 1.10 |

| Network |
| ------- |
| mainnet    |
| kovan    |

**Comment by dmichael** [comment]
I am seeing this as well on Kovan. Its not an issue with the specific network or version of the ethereum client being used (i tested several parity versions).

I traced it back to a call from the constructor of the order watcher to get the `net_version`. This call is failing in the Web3Wrapper due to some changes in how it is sending out the RPC requests. The request does not have an 'id' which causes it to error. More soon

**Comment by dmichael** [comment]
The error originates here
https://github.com/0xProject/0x-monorepo/blob/v2-prototype/packages/0x.js/src/0x.ts#L224
from `ZeroEx#createOrderWatcher`. This method calls `Web3Wrapper#getNetworkIdAsync ` (https://github.com/0xProject/0x-monorepo/blob/v2-prototype/packages/web3-wrapper/src/web3_wrapper.ts#L149) which sends a raw payload without an `id` field. This causes an empty response from the ethereum node

**Comment by dmichael** [comment]
Maybe just added something like `id: this._jsonRpcRequestId++` like its done in [this method](https://github.com/0xProject/0x-monorepo/blob/v2-prototype/packages/web3-wrapper/src/web3_wrapper.ts#L268) would solve it.

**Comment by dmichael** [comment]
Final thought - this lack of `id` in the payload is also affecting some of the other calls from the `Web3Wrapper`.

**Comment by dmichael** [comment]
Further: 

BAD
```
curl -X POST --data '{"jsonrpc":"2.0","method":"net_version"}' -H "Content-Type: application/json" http://localhost:8545
```

GOOD :)
```
curl -X POST --data '{"jsonrpc":"2.0","method":"net_version","id":777}' -H "Content-Type: application/json" http://localhost:8545
```

**Comment by LogvinovLeon** [comment]
Hi, @dmichael. Thanks for finding that. I've just pushed a [fix for that](https://github.com/0xProject/0x-monorepo/commit/fd9b3e0dcfcbfb15bdadd4c97d05e93a50b2a434). 

Unfortunately, it's Friday 4 pm and we don't think it's gonna be responsible for us to republish a bunch of packages before the weekend. For now you can either downgrade to the previous version or apply that patch to your local files to check that it's working.

We're gonna roll that fix out on Monday.

**Comment by dmichael** [comment]
Thanks @LogvinovLeon. The current release from master branch is `0.34` - a little behind the times but living on the edge can get you burned :fire:. Lots of good improvements in the codebase though and looking forward to a merge to master.

**Comment by LogvinovLeon** [comment]
Should be fixed with the @0xproject/web3-wrapper@0.7.0 release. Please reinstall your dependencies and make sure that the latest version of web3-wrapper gets picked up. (You might have a lock file).
Reopen if still valid.
